src/functions-and-arrays.js

```diff
@@ -52,15 +106,27 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(words) {
+  return words.filter((value, index)=> words.indexOf(value)===index);
+  }
+console.log(uniquifyArray(wordsUnique))
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
+function doesWordExist(value, array) {
+let status="false";
+for(let i=0; i<array.lenght; i++){
+  let word=array[i];
+  if(word==value){
+    status='true';
+    break;
+  }
+}return status;
+}
+console.log(doesWordExist('matter',wordsFind))
```

Comment:
Estaba muy bien enfocado, te falta revisar pequeños fallidos como el 'length'

```suggestion
 function doesWordExist(value, array) {
	let status = false;
	
	for(let i = 0; i < array.length; i++){
		let word = array[i];
		if (word === value) {
			status = true;
		}
	}
	
	return status;
 }
```